### PR TITLE
Fix: Store config in ~/.config/goft/config.yml

### PR DESCRIPTION
**Pull request by tkomatsu**
I've changed to store config file in ~/.config/goft/config.yml. I think it is easier for users to understand it is saved in the same location as other programs like GitHub CLI.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -49,10 +50,11 @@ all_platforms: vendor
...
+	mkdir -p $(CONFIG_FILE_DIR)
 	cp -i $(CONFIG_FILE_EXAMPLE) $(CONFIG_FILE_TARGET)
 	@echo "You need to edit $(CONFIG_FILE_TARGET) with your credentials"
 
 .PHONY: uninstall
 uninstall:
 	sudo rm $(INSTALL_PATH)/$(NAME)
-	rm $(CONFIG_FILE_TARGET)
\ No newline at end of file
+	rm $(CONFIG_FILE_TARGET)
```
Comment (mehdibo) [inline Makefile]: Don't you think it should also delete the directory we created?

File: cmd/root.go [hunk 2]
```diff
@@ -20,14 +19,14 @@ var (
...
-
 // NewRootCmd Create new root command
 func NewRootCmd() *cobra.Command {
 	cmd := cobra.Command{
 		Use:   "goft",
 		Short: "CLI tool to interact with 42's API",
 	}
-	cmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.goft.yaml)")
+	defaultconf := os.Getenv("HOME") + "/.config/goft/config.yml"
+	cmd.PersistentFlags().StringVar(&cfgFile, "config", defaultconf, "config file")
```
Comment (mehdibo) [inline cmd/root.go:29]: The flag's description should still say where the config path is located

**Review by mehdibo — COMMENTED** [review]
It looks good! just need to make the Makefile delete the created directory in `~/.config`

File: Makefile (hunk 1 above)
Comment (tkomatsu) [reply Makefile]: I should do that. I've fixed it.

File: cmd/root.go (hunk 2 above)
Comment (tkomatsu) [reply cmd/root.go:29]: `cobra` makes description with default value. We don't need to write default value in the description sentence.
```
$ goft -h
CLI tool to interact with 42's API

Usage:
  goft [command]

Available Commands:
  agu         Manage Anti Grav Units (AGUs)
  closes      Manage users' closes
  help        Help about any command
  requests    Send raw requests to the Intranet's API
  users       Interact with the user entity

Flags:
      --config string   config file (default "/Users/tkomatsu/.config/goft/config.yml")
  -h, --help            help for goft
  -v, --version         version for goft

Use "goft [command] --help" for more information about a command.
```

**Comment by tkomatsu** [comment]
Thank you for reviewing! I've fixed the Makefile.

**Comment by mehdibo** [comment]
Looks good, thank you!

**Pull request merged**